ui/src/shared/components/ResizeContainer.js

```diff
@@ -1,11 +1,22 @@
 import React, {PropTypes} from 'react'
 import ResizeHandle from 'shared/components/ResizeHandle'
+import classNames from 'classnames'
 
-const {node, string} = PropTypes
+const {node, number, string} = PropTypes
 
 const ResizeContainer = React.createClass({
   propTypes: {
     children: node.isRequired,
+    containerClass: string.isRequired,
+    minTopHeight: number,
+    minBottomHeight: number,
+  },
+
+  getDefaultProps() {
+    return {
+      minTopHeight: 200,
```

Comment:
suggest to factor these numbers out into `const` assignments just above, for consistency